MAINTAINERS

```diff
@@ -0,0 +1,16 @@
+Avocado-vt maintainers
+======================
+
+The intention of this file is not to establish who owns what portions of the
+code base, but to provide a set of names that developers can consult when they
+have a question about a particular subset and also to provide a set of names
+you can look for on github for any pull requests you might want to get approved.
+
+In general, if you have a question, you should send an email to the Virt Test
+development mailing list and not any specific individual privately.
+
+L: Avocado-devel <avocado-devel@redhat.com>
+M: Yanbing Du <ydu@redhat.com>
+M: Jiangang Wei <weijg.fnst@cn.fujitsu.com>
+M: Guannan Wayne Sun <gsun@redhat.com>
+M: Hao Liu <hliu@redhat.com>
```

Comment:
I think I could be replaced by xutian, as he’s more activated than me
recently.
On Apr 19, 2016 17:49, "Xu Tian" <notifications@github.com> wrote:

In MAINTAINERS
<https://github.com/avocado-framework/avocado-vt/pull/461#discussion_r60202358>
:

> +Avocado-vt maintainers
> +======================
> +
> +The intention of this file is not to establish who owns what portions of the
> +code base, but to provide a set of names that developers can consult when they
> +have a question about a particular subset and also to provide a set of names
> +you can look for on github for any pull requests you might want to get approved.
> +
> +In general, if you have a question, you should send an email to the Virt Test
> +development mailing list and not any specific individual privately.
> +
> +L: Avocado-devel <avocado-devel@redhat.com>
> +M: Yanbing Du <ydu@redhat.com>
> +M: Jiangang Wei <weijg.fnst@cn.fujitsu.com>
> +M: Guannan Wayne Sun <gsun@redhat.com>
> +M: Hao Liu <hliu@redhat.com>

@ldoktor <https://github.com/ldoktor> view from your list most guys are
expert in libvirt side,t hey will help answer question and help resolve
bugs issue in libvirt side, so you @ldoktor <https://github.com/ldoktor>
and @clebergnu <https://github.com/clebergnu> will focus on qemu-kvm side,
right ?

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/avocado-framework/avocado-vt/pull/461/files/2a3c72bcbe1371a865610e10fa31b87919292eea#r60202358>